### PR TITLE
Submitting M-Files Online Client 23.4.12528.8

### PR DESCRIPTION
**Pull request by aurel-bul**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106311)

### CODE REVIEW

**Comment by aurel-bul** [comment]
@microsoft-github-policy-service agree

File: manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.locale.eng.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: M-FilesCorporation.M-FilesOnline
+PackageVersion: 23.4.12528.8
+PackageLocale: eng
+Publisher: M-Files Corporation
+PackageName: M-Files Online
+License: © 2022 M-Files, All Rights Reserved
```
Comment (mdanish-kh) [inline manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.locale.eng.yaml:9]:
```suggestion
License: Proprietary
```
A copyright isn't a license

File: manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.locale.eng.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: M-FilesCorporation.M-FilesOnline
+PackageVersion: 23.4.12528.8
+PackageLocale: eng
```
Comment (mdanish-kh) [inline manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.locale.eng.yaml:6]:
```suggestion
PackageLocale: en-US
```
Please use the BCP-47 language codes.

File: manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: M-FilesCorporation.M-FilesOnline
+PackageVersion: 23.4.12528.8
+DefaultLocale: eng
```
Comment (mdanish-kh) [inline manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.yaml:6]:
```suggestion
DefaultLocale: en-US
```
Please use the BCP-47 language codes.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 
> 'M-FilesCorporation.M-FilesOnline.locale.eng.yaml', expected 
> 'M-FilesCorporation.M-FilesOnline.locale.en-US.yaml'  

The filename has a `g` instead of `-US`.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**